src/lib/Behat/PageElement/UniversalDiscoveryWidget.php

```diff
@@ -55,12 +55,21 @@ function () use ($expectedContentName) {
             });
 
         if ($this->isMultiSelect()) {
-            $itemToSelect = $this->context->getElementByText($expectedContentName, sprintf($this->fields['elementSelector'], $depth));
-
-            $this->context->findElement($this->fields['tabSelector'])->mouseOver();
-            $itemToSelect->mouseOver();
-            $this->context->waitUntilElementIsVisible($this->fields['selectContentButton'], $this->defaultTimeout, $itemToSelect);
-            $this->context->findElement($this->fields['selectContentButton'], $this->defaultTimeout, $itemToSelect)->click();
+            for ($i = 0; $i < 3; ++$i) {
```

Comment:
Would it make sense to add a sleep(1) after each loop? Usually retry mechanisms are implemented that way.

Reply:
I'd prefer not - if you look at actions taken, it looks like:
1. find the item to select
2. mouse over the first UDW tab
3. mouse over the item found
4. wait until select button in the item element is visible
5. click the select button
if at any point we'll get the exception, we repeat from the first step. Plus, we have the built-in 5 seconds timeout in steps 1, 2 and 5. I don't think we'll benefit from this additional second.